src/aap_eda/services/project/imports.py

```diff
@@ -61,13 +61,12 @@ def wrapper(self: ProjectImportService, project: models.Project):
         try:
             func(self, project)
             project.import_state = models.Project.ImportState.COMPLETED
-        except Exception as e:
+        except ScmEmptyError as e:
             # if a project is empty, sync status should show completed
-            project.import_state = (
-                models.Project.ImportState.COMPLETED
-                if "Project folder is empty" in e.args[0]
-                else models.Project.ImportState.FAILED
-            )
+            project.import_state = models.Project.ImportState.COMPLETED
```

Comment:
should be this failed? in the description you say: 
> This fix identifies the first case and set the state to be failed.

Reply:
ScmEmptyError is for the second case where the repo is truly empty. We want it to be completed.